validator/remote/src/main/java/tech/pegasys/teku/validator/remote/apiclient/ResponseHandler.java

```diff
@@ -64,15 +69,22 @@ public ResponseHandler(final JsonProvider jsonProvider, final Class<T> responseC
         .handleResponse(request, response);
   }
 
-  private Optional<T> unknownResponseCodeHandler(final Request request, final Response response)
-      throws IOException {
+  private Optional<T> unknownResponseCodeHandler(final Request request, final Response response) {
+    LOG.debug(
+        "Unexpected response from Beacon Node API (url = {}, status = {}, response = {})",
+        request.url(),
+        response.code(),
+        response.body());
     throw new RuntimeException(
         String.format(
-            "Unexpected response from Beacon Node API (url = %s, status = %s, response = %s)",
-            request.url(), response.code(), response.body().string()));
+            "Unexpected response from Beacon Node API (url = %s, status = %s)",
+            request.url(), response.code()));
   }
 
   private Optional<T> noValueHandler(final Request request, final Response response) {
+    if (response.code() != SC_ACCEPTED && response.code() != SC_NO_CONTENT) {
+      LOG.warn("No value from url={}, status={}", request.url(), response.code());
```

Comment:
I think this warning helps but "No value" isn't really what's going on.  It's really a server error.